wsmaster/che-core-api-auth-bitbucket/src/main/java/org/eclipse/che/security/oauth/BitbucketOAuthAuthenticatorProvider.java

```diff
@@ -70,23 +76,26 @@ private OAuthAuthenticator getOAuthAuthenticator(
         && !isNullOrEmpty(tokenUri)
         && Objects.nonNull(redirectUris)
         && redirectUris.length != 0) {
+      String trimmedOauthEndpoint = trimEnd(oauthEndpoint, '/');
+      authUri =
+          trimmedOauthEndpoint.equals("https://bitbucket.org")
+              ? authUri
+              : trimmedOauthEndpoint + "/rest/oauth2/1.0/authorize";
+      tokenUri =
+          trimmedOauthEndpoint.equals("https://bitbucket.org")
+              ? tokenUri
+              : trimmedOauthEndpoint + "/rest/oauth2/1.0/token";
```

Comment:
can we move it to a separate method, also please add constants for endpoints

Reply:
Simplified by moving the `trimmedOauthEndpoint.equals("https://bitbucket.org")` check to a local variable.